topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.md

```diff
@@ -637,26 +686,23 @@ Let's go ahead and classify those sequences using the Bayesian classifier with t
 > {: .question}
 {: .hands_on}
 
-Also of note is that *unknown* only pops up as a classification if the classifier cannot classify your
-sequence to one of the domains.
-
-At this point we have curated our data as far as possible and we're ready to see what our error rate is.
+The data is now as clean as we can get it.  Despite that, our remaining sequences will still have errors in them.
```

Comment:
This is a demotivating last sentence in a chapter ... :(
Can we reformulate this to sound a little bit more positive, or leave it out?